src/contentsharecontroller/ContentShareMediaStreamBroker.ts

```diff
@@ -66,7 +66,11 @@ export default class ContentShareMediaStreamBroker implements MediaStreamBroker
     return {
       audio:
         !sourceId && new DefaultBrowserBehavior().getDisplayMediaAudioCaptureSupport()
-          ? true
+          ? {
+              echoCancellation: false,
+              noiseSuppression: false,
```

Comment:
Just want to point out that it seems we use `goog` prefix for the audio stream https://github.com/aws/amazon-chime-sdk-js/blob/main/src/devicecontroller/DefaultDeviceController.ts#L1303. 
I think Google has migrated to use the standard one so this is probably fine but wonder if we need to update the other code later too.